accesscontroltool-install-packages/accesscontroltool-cr-package/pom.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
please don't use abbreviations no one will understand. Why not accesscontroltool-cloud-package? The official name is now AEM in the cloud!

Reply:
`cr` stands for cloud-ready, but yes, `accesscontroltool-cloud-package` is probably better to understand, will adjust it.